packages/core/src/api/apis/default/index.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This folder contains definitions for all core APIs.
```

Comment:
implementations*? If I understand this correctly this would be a folder of default implementations for the APIs? Which makes sense imo, def better than the current solution :p

Reply:
right, i can change the name, didn't love default but didn't have a strong name i could think of

Reply:
Ah no I just meant to correct the comment, `definitions -> implementations`

Didn't have any opinion on the folder name but now that you mention it `implementations` or `impls` might be better. The rename is really simple to do though as we're reexporting things anyway, so only a small change within this package

Reply:
done